### PR TITLE
[SC-4973] rename FEATURE_ADMIN_TOGGLE_STUDENT_VISIBILITY

### PR DESCRIPTION
**Pull request by pawelchoniawko**
# Description
The new name can be ADMIN_TOGGLE_STUDENT_VISIBILITY
The renaming should happen in client and server.

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-4973

### CODE REVIEW

File: config/default.schema.json [hunk 1]
```diff
@@ -163,7 +163,7 @@
 			"pattern": ".*(?<!/)$",
 			"description": "Content API URL of ghost blog (should end without a slash)"
 		},
-		"FEATURE_ADMIN_TOGGLE_STUDENT_VISIBILITY": {
```
Comment (CordlessWool) [inline config/default.schema.json:166]: please keep it as feature_...._enabled and switch to true, false logic

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

File: controllers/administration.js [hunk 2]
```diff
@@ -2548,7 +2548,7 @@ const schoolFeatureUpdateHandler = async (req, res, next) => {
 		delete req.body.videoconference;
 
 		// Toggle teacher's studentVisibility permission
-		const studentVisibilityFeature = Configuration.get('FEATURE_ADMIN_TOGGLE_STUDENT_VISIBILITY');
+		const studentVisibilityFeature = Configuration.get('ADMIN_TOGGLE_STUDENT_VISIBILITY');
```
Comment (j-jmnz) [inline controllers/administration.js:2551]: This should also be changed to the new name

File: views/administration/school.hbs [hunk 3]
```diff
@@ -220,7 +220,7 @@
 							<canvas id="logo-canvas" width="35" height="35" class="hidden"></canvas>
 						</div>
 
-						{{#ifConfig "FEATURE_ADMIN_TOGGLE_STUDENT_VISIBILITY" "enabled"}}
+						{{#ifConfig "ADMIN_TOGGLE_STUDENT_VISIBILITY" "enabled"}}
```
Comment (j-jmnz) [inline views/administration/school.hbs:223]: Also this should be changed to the new name, and should look for `true` instead of `enabled`

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 25
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Comment by Parnswir** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5756363&bid=18843217)

**Pull request merged**